pandas/tests/tools/test_to_datetime.py

```diff
@@ -1765,7 +1765,7 @@ def test_unit_array_mixed_nans(self, cache):
         result = to_datetime(values, unit="D", errors="ignore", cache=cache)
         expected = Index(
             [
-                11111111,
+                Timestamp(11111111, unit="D"),
```

Comment:
isnt the point of this test to have mixed types?

Reply:
yup, good one, thanks - I've changed the input to be a bigger one which forces it to raise